src/components/grids/TabbedDisplay/index.tsx

```diff
@@ -22,13 +19,15 @@ export type TabbedDisplayStyleSpec = {
   inactive: TabStyle;
   active: TabStyle;
   hover: TabStyle;
+  tabLabel: CSSProperties;
```

Comment:
By using `CSSProperties`, we are allowing the user to potentially provide styles that conflict with other overrides (`textColor`, for instance). Maybe we should just add `tabFontSize`?